lib/class-wp-block-patterns-registry.php

```diff
@@ -1,14 +1,14 @@
 <?php
 /**
- * Blocks API: WP_Patterns_Registry class
+ * Blocks API: WP_Block_Patterns_Registry class
  *
  * @package Gutenberg
  */
 
 /**
  * Class used for interacting with patterns.
  */
-final class WP_Patterns_Registry {
+final class WP_Block_Patterns_Registry {
```

Comment:
Not directly related to the renaming, but I wonder here about pluralization, specifically since there's already precedent in core with using the singular form for a "registry of many X", i.e. `WP_Block_Type_Registry`. Should these not be consistent? (`WP_Block_Pattern_Registry`)

**Edit:** After this, I realize there is `WP_Block_Styles_Registry` as well, so inconsistency already exists unfortunately 😞

Reply:
Do you have a preference knowing the inconsistency exists?

Reply:
Looking through some of the other core classes for prior art, I do also see there's another example of the plural form with `WP_Dependencies` (`WP_Scripts`, `WP_Styles`), which is another sort of "registry of items". Maybe it's best to lean into this, and consider `WP_Block_Type_Registry` as the exception.